### PR TITLE
Access lvl for admins

### LINKED ISSUE #3717: Incorrect access shown and saved when admin edits content set to a user's ACL. (Trac #3717)
_Original ticket http://trac.elgg.org/ticket/3717 on 41600552-06-29 by trac user slyhne, assigned to unknown._

Elgg version: 1.8 Beta

One thing that alway bothered me is that admin can't see what private access collection a user has set for his/hers content.
futhermore if an administrator edited any content, it would allways be set to access PRIVATE.

I have solved this in (attached to this ticket):
/views/default/input/access.php
/views/default/output/access.php

If an administrator is looking at eg. a users blog, he can see the name of the access collection it's saved under.

If an administrator edits a users blog (does happen from time to time), he can preserve whatever access collection the user has set.

This change also need a language string like:

    'access:userdefined' => "Userdefined access",

### PR DESCRIPTION
**Pull request by jdalsem**
fixes #3717

### CODE REVIEW

File: views/default/input/access.php [hunk 1]
```diff
@@ -48,6 +48,17 @@
 }
 
 if (is_array($vars['options_values']) && sizeof($vars['options_values']) > 0) {
+
+	if ($vars['value'] && !array_key_exists($vars['value'], $vars['options_values']) && $container && $container->canEdit()) {
```
Comment (mrclay) [inline views/default/input/access.php]: I wonder if this code will cause a nasty merge conflict with #7766 (now in 1.x).

Comment (jdalsem) [inline views/default/input/access.php]: i think so, i can move the PR to 1.x

Comment (jdalsem) [inline views/default/input/access.php]: do you otherwise agree with the changes ?

Comment (mrclay) [inline views/default/input/access.php]: Yeah, it's good

Comment (jdalsem) [inline views/default/input/access.php]: New PR here #7898, you already fixed part of the problem, so no need for the changes to input/access anymore

**Pull request closed without merging**